kolibri/content/serializers.py

```diff
@@ -76,6 +78,20 @@ class ContentNodeListSerializer(serializers.ListSerializer):
 
     def to_representation(self, data):
 
+        # Dealing with nested relationships, data can be a Manager,
+        # so, first get a queryset from the Manager if needed
+        data = data.all() if isinstance(data, Manager) else data
+
+        data = data.prefetch_related(
+            'assessmentmetadata',
+            'files',
+        ).select_related('license')
+
+        cache_key = 'contentnode_list_{db}_{query}'.format(db=get_active_content_database(), query=hash(str(data.query)))
+
+        if cache.get(cache_key):
+            return cache.get(cache_key)
```

Comment:
Yeah, always valid unless the content database is updated - we are only caching responses that only contain topics here, so there is no progress annotation on them.